packages/runtime/src/dataViews/consumption/LocalAttributeDVO.ts

```diff
@@ -47,6 +47,8 @@ export interface SharedToPeerAttributeDVO extends LocalAttributeDVO {
     sourceAttribute: string;
     isOwn: true;
     tags: string[];
+    deletionDate?: string;
+    status?: string;
```

Comment:
Is there a reason to not use a nested object `deletionStatus: { "deletetionDate": ..., "status": ... }`.

If we can't do this I feel like `status` should receive a more suggestive name.

Reply:
We don't use a nested object for `shareInfo`, so I thought we shouldn't use one for `deletionStatus`, neither. But, I agree with you that the name `status` then isn't ideal. How about `deletionInfo: {"deletionStatus": ..., "deletionDate": ...}`?

Reply:
Sounds good but I'd like for `LocalAttributeDeletionStatus` to match this structure, so their correspondence will be immediately apparent. You could just rename `LocalAttributeDeletionStatus` to `LocalAttributeDeletionInfo`, its `status` field to `deletionStatus` and the `LocalAttribute`'s field to `deletionInfo` (which would nicely match `shareInfo`).

Reply:
This is exactly what I meant :)